src/main/kotlin/uk/gov/gdx/datashare/controllers/AcquirersController.kt

```diff
@@ -85,6 +85,36 @@ class AcquirersController(
     acquirerRequest: AcquirerRequest,
   ) = acquirersService.addAcquirer(acquirerRequest)
 
+  @DeleteMapping
+  @Operation(
+    summary = "Delete Acquirer",
+    description = "Need scope of events/admin",
```

Comment:
Our documentation is autogenerated from this, so I think it would be nice to have a description of some of the logic. Under what circumstances does this delete what?